src/components/badge/index.tsx

```diff
@@ -230,38 +236,35 @@ class Badge extends PureComponent<BadgeProps> {
     }
   }
 
+  renderCustomElement() {
+    const {customElement} = this.props;
+    return customElement && <View>{customElement}</View>;
```

Comment:
Is wrapping the customElement with a View necessary?

Reply:
Nope.. fixed :)